filter:
  state_pool_size: 1000
  number_of_particle: 200
predictor:
  maximal_dead_recknoning_travelled_distance: 1.
  maximal_dead_recknoning_elapsed_time: 1.
twist_updater:
  minimal_rate: 10.0
pose_updater:
  minimal_rate: 1.0
  trigger: once
range_updater:
  minimal_rate: 10.0
  trigger: always
leader_twist_updater:
  minimal_rate: 10.0


